src/Interface/HasOrder/Instance.agda

```diff
@@ -23,3 +23,13 @@ instance
     ⊎.[ <⇒≤ , ≤-reflexive ] }
   ℤ-hasPartialOrder = HasPartialOrder ∋ record { ≤-antisym = Int.≤-antisym }
   ℤ-hasDecPartialOrder = HasDecPartialOrder {A = ℤ} ∋ record {}
+
+  open import Data.Rational using (ℚ)
+  import Data.Rational.Properties as Rat hiding (_≟_)
+
+  ℚ-Dec-≤ = ⁇² Rat._≤?_
+  ℚ-Dec-< = ⁇² Rat._<?_
```

Comment:
Can you add a TODO to move these back to `agda-stdlib-classes` at some point?

Reply:
No: https://github.com/omelkonian/agda-stdlib-classes/pull/1